Chp6Lsn3/Chp6Lsn3/Program.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chp6Lsn3
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            int initialValue = 0;
+            int[] N = UsesrInput(initialValue);
+            int[] endValues = SmallestAndLargest(N);
+            PrintSequence(endValues);
+        }
+
+        /// <summary>
+        /// Takes in a series of values provided by the user.
+        /// </summary>
+        /// <param name="iV">Initial value for initial values sake.</param>
+        /// <returns>An array created by the user.</returns>
+        public static int[] UsesrInput(int iV)
+        {
+            int isInput;
+            bool resolved = false;
+            int[] userSeries = new int[2];
+            int i = 0;
+            int m = 1;
+            Console.WriteLine("Please enter a series of positive integers. When finished enter the word 'done'.");
+
+            while (resolved == false)
+            {
+                int numColumns = userSeries.GetLength(0);
+                string input = Console.ReadLine();
+                bool isInt = System.Int32.TryParse(input, out isInput);
+
+                if (isInt)
+                {
+                    if (m > numColumns)
```

Comment:
The variable 'm' is not necessary.  You could simply write:

    if (i + 1 > numColumns)

However, this is not important; it does not need to be changed.